{"ast":null,"code":"function catchErrors(error, callbackError) {\n  let errorMessage;\n\n  if (error.response) {\n    //the request was made and the server respond with a status code that is not in the range of 2XX\n    errorMessage = error.response.data;\n    console.error(\"Error request\", errorMessage);\n  } else if (error.request) {\n    //the request was made but no responde was receive \n    errorMessage = error.request;\n    console.error(\"Error request\", errorMessage);\n  } else {\n    //something else happend in making the request that triggered an error\n    errorMessage = error.message;\n    console.error(\"Error request\", errorMessage);\n  }\n\n  callbackError(errorMessage);\n}\n\nexport default catchErrors;","map":null,"metadata":{},"sourceType":"module"}